bioblend/galaxy/workflows/__init__.py

```diff
@@ -632,13 +632,25 @@ def show_invocation(self, workflow_id, invocation_id):
         url = self._invocation_url(workflow_id, invocation_id)
         return self._get(url=url)
 
-    def get_invocations(self, workflow_id):
+    def get_invocations(self, workflow_id, history_id=None, user_id=None, include_terminal=True, limit=None, view='collection', step_details=False):
```

Comment:
What about leaving this method as it is and add a note here suggesting to use `InvocationClient.get_invocations()` for more advanced filtering?

Reply:
is there a reason we shouldn't provide it?

Reply:
Because I'd like to keep the invocation methods here only for compatibility with Galaxy < 19.09, with all new features going to `InvocationClient`.

Reply:
Ah, ok

Reply:
Should `gi.workflows.get_invocations()` be considered deprecated then?

I still find it a handy way to filter invocations produced by a particular workflow. But I guess when this is merged it can now be done by `gi.invocations.get_invocations(workflow_id=id)` as described above.

Reply:
@simonbray Yep, that's the idea!